autorest/adal/config.go

```diff
@@ -79,3 +79,50 @@ func NewOAuthConfig(activeDirectoryEndpoint, tenantID string) (*OAuthConfig, err
 		DeviceCodeEndpoint: *deviceCodeURL,
 	}, nil
 }
+
+// NewOAuthConfigWithAPIVersion returns an OAuthConfig with tenant specific urls.
+// If apiVersion is not nil the "api-version" query parameter will be appended to the endpoint URLs with the specified value.
```

Comment:
can we make the old function call this one to avoid duplicated code ?

Reply:
I presume we would call it with the 1.0 param

Reply:
Derp yep will fix.